### DISCUSSION TITLE
GUI RAW-Editor Featurerequest

### DISCUSSION THREAD

**Discussion by sohnvonoff** [home-assistant/frontend#10062] [Accessibility] [answered]
Can the operating functionality of the GUI RAW editor be designed similar to the integration of the 'FILE editor'?
It would be important:
- Show space as period (very impotant!!!!)
- collapse/expand logical blocks
- Save comments

Thanks
Bernd

**Comment by spacegaier** [comment, marked as answer]
Reg. comments: That is not possible, see https://github.com/home-assistant/frontend/issues/8203

**Reply by sohnvonoff** [reply]
OK,
But, the period and the logical blocks. 
That's actually just a problem of representation. It doesn't have to be saved. This feature would make the processing of complex structures much easier.
